editor/src/components/canvas/controls/comment-indicator.tsx

```diff
@@ -443,7 +447,7 @@ const HoveredCommentIndicator = React.memo((props: HoveredCommentIndicatorProps)
         background: colorTheme.bg1.value,
         zIndex: 1,
         position: 'fixed',
-        top: position.y,
+        bottom: canvasHeight - IndicatorSize - position.y,
```

Comment:
this isn't in the part of the canvas that zooms (so should be zoom-safe) - right?

Reply:
this is happily unaffected by zoom :)